src/blocks/ui/insights/personalization.js

```diff
@@ -37,6 +37,21 @@ const Personalization = ( {
 		};
 	}, [ clientId ] );
 
+	const ranges = [
```

Comment:
This seems to more or less duplicate the periods object we already have, in addition the labels here are not translatable. It would be better to reuse the periods object, and make the `DateRange` component work with the same values e.g. `P7D`, `P30D` and `P90D`

Reply:
@roborourke I notice the periods object is located within a [`.ts` file](https://github.com/humanmade/aws-analytics/blob/master/src/data/periods.ts). I suspect this can't be imported to a [`.js` file](https://github.com/humanmade/aws-analytics/blob/master/src/blocks/ui/insights/personalization.js) and I would need to recreate a new object is that correct?

Reply:
Ah right yes, we have the split build processes still... my usual approach would just be to try it out and see but I think you're right.

In which case it might be good to make a non typescript copy of the periods object and use that, with a comment noting that it should use the typescript version in future.

Reply:
@roborourke I got this setup and running but before I push my current changes up, given that the `DateRange`  component is only being used once and would essentially be replaced at later stage, would it not be better to keep the duplicate and get it to work rather than duplicating the typescript copy?